docs/source/index.rst

```diff
@@ -1,7 +1,7 @@
 Genestack Python Client Library
 ===============================
 
-The Genestack Python Client Library is a Python library that allows you to interact programatically with an instance of the Genestack platform.
+The Genestack Python Client Library is a Python library that allows you to interact programmatically with an instance of the Genestack platform.
 
 **Python 2** supported from version `2.7.5`, **Python 3** is not supported.
```

Comment:
While we're _fixing gramma_, make that line
```rst
We support **Python 2** from version *2.7.5* and newer, **Python 3** is not supported.
```
(backticks are n/a here, that's not monospace text in rST)